pimd/pim_cmd.c

```diff
@@ -11200,20 +9851,17 @@ void pim_cmd_init(void)
 	install_element(CONFIG_NODE, &no_debug_pim_mlag_cmd);
 	install_element(CONFIG_NODE, &debug_pim_vxlan_cmd);
 	install_element(CONFIG_NODE, &no_debug_pim_vxlan_cmd);
-#if PIM_IPV != 6
```

Comment:
why is this removed?

Reply:
oh ok ok got it, since we have made the files itself separate so its no problem.